src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -2150,15 +2161,6 @@ Debug("GenerateVoidILStartupMethod: Linux: Setting the PInvoke native profiler l
     return hr;
   }
 
-  ULONG string_len = 0;
```

Comment:
Another unrelated change, `string_contents` was never used anywhere so I deleted. It was probably used for debugging at some point